fastapi/routing.py

```diff
@@ -366,10 +373,7 @@ def __init__(
             self.secure_cloned_response_field = None
         self.status_code = status_code
         self.tags = tags or []
-        if dependencies:
-            self.dependencies = list(dependencies)
-        else:
-            self.dependencies = []
+        self.dependencies = list(dependencies or [])
```

Comment:
Could you please amend it like below?

```py
self.dependencies = list(dependencies) or []
```

Reply:
Sure

Reply:
Yeah.

Reply:
```py
self.dependencies = dependencies if dependencies else []
```
I think this is more clear.

Reply:
Or maybe
```py
self.dependencies = dependencies or []
```

Reply:
This looks better, thanks!

I had to do a minor update because of this change: 
This `dependencies` are now any kind `Sequence`, and `dependencies.copy().extend(...)` only works with lists.
I have updated it to `list(dependencies).extend(...)` instead

Reply:
Why do you make new commit instead of update the one and make force push?

Reply:
When I'm the reviewer I find it easier if my comments are addressed in separate commits, so that I don't need to read all the changes again.

However that is a matter of personal preference, I will squash everything in a single commit for you

Reply:
I just do want to be available reduntant changes in the history. I am not fimiliar with the FastApi convention for this topic. I encourage you to follow the convention of the FastApi.

Reply:
Thanks @cikay for the interest here!

Nevertheless, there's no need to request changes that depend more on taste. I actually prefer the original proposed version, so I'm gonna revert it to that state.

Also, no need to worry about commits, which ones, how many, or the messages... I'll squash everything and update the message when merging.